### ISSUE TITLE
Middle mousewheel stops working after logid exits

### ISSUE THREAD

**Issue by gdiShun** [PixlOne/logiops#123]
Pretty much the title. If you have `target: true` and use logid to bind the mousewheel, when it crashes or exits, the mousewheel will stop working entirely. Restarting logid is necessary to re-enable it. It doesn't hand it back to the OS I guess you could say.

**Comment by PixlOne** [comment]
Yeah, this a bit of a known bug. Restarting the mouse works as a workaround. I'll need to handle SIGINT in order to fix this. This will be impossible to fix if logid crashes though.
